lib/components/ActionTrigger/ActionTrigger.module.scss

```diff
@@ -19,27 +19,23 @@ $line-height: 4.5*$grid-size;
         }
     }
 
-    a > & {
-        color: var(--color-hyperlink-rest);
-    }
-
     a:hover > & {
-        color: var(--color-hyperlink-hover);
-        background-color: var(--color-bg-hover);
+        color: var(--color-accent-hover);
+        background-color: var(--color-state-hover);
     }
 
     a:focus > & {
         outline-offset: -1px;
-        outline: 1px dashed var(--color-border-focus);
-        color: var(--color-hyperlink-focus);
+        outline: 1px dashed var(--color-accent);
 
         &:not(.disabled) {
-            background-color: var(--color-bg-hover);
+            color: var(--color-accent-hover);
+            background-color: var(--color-state-hover);
         }
     }
 
     &.disabled {
-        color: var(--color-text-disabled);
+        color: var(--color-foreground-disabled);

```

Comment:
are those color identifiers coming from some agreement with the design team so we are on the same page?

Reply:
yes, I have been working with Arturo to get this set of colors. Once this is in renderer I will continue to work with him to have them also in their design toolkit